diamm/templates/website/archive/archive_list.jinja2

```diff
@@ -1,14 +1,17 @@
 {% extends "base.jinja2" %}
 
 {% block body %}
-    <div>
+    <div class="row">
+        <div class="sixteen columns">
         <h1>Archives</h1>
-        {% for archive in content.results %}
-        <div class="search-result">
-            <h4><a href="{{ archive.url }}">{{ archive.name }}</a></h4>
-            <a href="{{ archive.city.url }}">{{ archive.city.name }}</a>, {{ archive.city.country }}
+            <p>Note: Archives are sorted by city name.</p>
```

Comment:
Just to confirm: given that so many of the archive names begin with some form of Archive or Bibliotheque, this is the way we want to keep the sorting, eh?